server/.env

```diff
@@ -0,0 +1,11 @@
+# DB_HOST=db.tchaegbogyejdhrymbni.supabase.co
```

Comment:
This file should be in your .gitignore, so secrets like passwords don't get committed to git